### DISCUSSION TITLE
Package related query

### DISCUSSION THREAD

**Discussion by shikhar229169** [Cyfrin/foundry-full-course-cu#2001] [Q&A] [answered]
What was the name of the package that was used to produce pattern like below:
```
/////////////////////////////////
////////// Text /////////////////
/////////////////////////////////
```
I just performed a reset on my Ubuntu and forgot the name of this package.

**Comment by EngrPips** [comment, marked as answer]
Hello @shikhar229169, here you go `https://github.com/transmissions11/headers`

**Reply by shikhar229169** [reply]
Thank You

**Reply by EngrPips** [reply]
You are welcome.
